site/docs/tools/iceberg/spark.md

```diff
@@ -301,13 +301,13 @@ Spark/Hadoop configs.
 # read from branch dev
 spark.read().format("iceberg").load("testing.region@dev")
 # read specifically from hash
-spark.read().format("iceberg").load("testing.region@<hash>")
+spark.read().format("iceberg").load("testing.region#<hash>")
 
 spark.sql("SELECT * FROM nessie.testing.`region@dev`")
-spark.sql("SELECT * FROM nessie.testing.`region@<hash>`")
+spark.sql("SELECT * FROM nessie.testing.`region#<hash>`")
```

Comment:
nit: maybe one more example for `region@dev#<hash>` (also for Flink/Hive)?

Reply:
`table@reference#hash` would be nice to have for the same of completeness, because detached hash references (`table#hash`) have authorization implications (may not be allowed when `table@reference#hash` is allowed).

Reply:
I was actually thinking about adding another example for `region@dev#<hash>` after seeing #7565

Let me add it now that you are also suggesting to do it.